src/shared/components/cancerSummary/CancerSummaryContent.tsx

```diff
@@ -174,20 +182,21 @@ export class CancerSummaryContent extends React.Component<ICancerSummaryContentP
     }
 
     private getColors(color:string) {
-        const colors:{[id:string]:string} = {
+        const alterationToColor: Record<keyof ICancerTypeAlterationPlotData, string> = {
```

Comment:
there's no reason for this map to be rebuilt on every call

Reply:
@computed?